### PR TITLE
[New Exercise]: Perfect-numbers

### PR DESCRIPTION
**Pull request by meatball133**

### CODE REVIEW

File: exercises/practice/perfect-numbers/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "authors": ["meatball133"],
```
Comment (ErikSchierboom) [inline exercises/practice/perfect-numbers/.meta/config.json]: Just a little FYI. This formatting is not the one used by `configlet fmt`. That is totally fine, but if you ever want to enable `configlet fmt` linting, this is something to consider.

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/practice/perfect-numbers/PerfectNumbers.tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        It "Edge case (no factors other than itself) is classified correctly" {
+            $got = Invoke-PerfectNumbers -Number 1
+            $want = "deficient"
+
+            $got | Should -BeExactly $want
+        }
+    }
+
+    Context "Failing Tests" {
```
Comment (ErikSchierboom) [inline exercises/practice/perfect-numbers/PerfectNumbers.tests.ps1]: This seems to describe the implementation, not the context. Maybe:

```suggestion
    Context "Invalid Inputs" {
```
